### ISSUE TITLE
no ouput

### ISSUE THREAD

**Issue by ReenigneArcher** [steffen-karlsson/stalesweeper#30]
Using dryrun mode, there is no output.

https://github.com/LizardByte/community/actions/runs/10965339345/job/30450926954

The only thing in the logs is:
```txt
> Run steffen-karlsson/stalesweeper@v1.1.1
  with:
    repo-token: ***
    message: It seems this discussion hasn't had any activity in the past 90 days. To help keep our backlog tidy, we'll be closing this discussion. If it's still something you'd like addressed, please let us know by leaving a comment. Thanks!
  
    days-before-close: 90
    close-unanswered: true
    category: Sunshine
    close-reason: OUTDATED
    dry-run: true
    verbose: true
```

There are ~500 discussions available, so I would expect something.

I am using a PAT, which has the `repo` scope:
![image](https://github.com/user-attachments/assets/74faad08-ba5b-44ac-8b52-90704f552e46)

I also tried giving it the `write:discussion` and `read:discussion` scope, but I think those are outdated scoped from when "teams" could have discussions.
![image](https://github.com/user-attachments/assets/6b25ba96-8456-4632-b935-15b8df008f35)

Here is my workflow: https://github.com/LizardByte/community/blob/master/.github/workflows/discussions-stale.yml

Any ideas what's wrong here?

**Comment by thenick775** [comment]
I got confused by this as well, but in reading the code its intentional (no discussions are fetched).

I've changed this in my fork over [here](https://github.com/thenick775/stalesweeper) in case your interested

**Comment by steffen-karlsson** [comment]
@thenick775 please feel free to open a PR and Ill happily look at it 👍

**Comment by ReenigneArcher** [comment]
@steffen-karlsson are you planning to maintain this action?

Last commit was almost 2 years ago, and even depdendabot is paused due to inactivity.

**Comment by steffen-karlsson** [comment]
Yes I'm @ReenigneArcher , haven't had notifications on issues and sorry for that!

But it has been working flawlessly for us.

**Comment by thenick775** [comment]
> [@thenick775](https://github.com/thenick775) please feel free to open a PR and Ill happily look at it 👍

I may allow others here to pull my logic back to this repo as a heads up, I've made some significant changes and dependency updates, and it might not translate back cleanly now!
